deployment/lambdas/submit_job/lambda_entrypoint.py

```diff
@@ -361,7 +361,7 @@ def get_submission_data(tumor_sample_md, normal_sample_md, subject_id, api_auth)
     tumor_sv_vcf = get_file_path(fr'^.+{tumor_id}.sv.vcf.gz$', file_list)
 
     # Set output directory using tumor BAM path
-    outdir_regex = re.compile(r'^gds://[^/]+/(.+)/wgs_tumor_normal/.+/(L\d+)_(L\d+)_dragen/.+?\.bam$')
+    outdir_regex = re.compile(r'^gds://[^/]+/(.+)/wgs_tumor_normal/.+/(L\d+)_(L\d+)_dragen_somatic/.+?\.bam$')
```

Comment:
Not blocker / Just saying

Future recommendation
While this approach work, understood that GPL might go away at some point; it won't support backward compatibility with older somatic run output. Consider, if GPL to (re) run with an older subject/case that still bears now obsolete somatic run output structure.

Portal has solve this in liborca module for parsing workflow output of every generations of workflow run by building well-known output lookup keys.

https://github.com/umccr/data-portal-apis/blob/dev/data_processors/pipeline/tools/liborca.py

Please also see related discussion / convo in PR
https://github.com/umccr/data-portal-apis/pull/578

If liborca is useful or reusable for the case, it could get package it in libumccr, if any. Suggestion/discussion welcome.

Reply:
Good point, thanks - added minimal backward compatibility support in [db77aad](https://github.com/umccr/gridss-purple-linx-nf/pull/31/commits/db77aad748cf3505a6cdd62bf18ca6dc5a7c22f9).

Reply:
Nice!